test/e2e/suites/import-gitops-v3/import_gitops_v3_test.go

```diff
@@ -42,16 +42,16 @@ var _ = Describe("[Docker] [Kubeadm] - [management.cattle.io/v3] Create and dele
 			ClusterctlConfigPath:           flagVals.ConfigPath,
 			ClusterctlBinaryPath:           e2eConfig.GetVariable(e2e.ClusterctlBinaryPathVar),
 			ArtifactFolder:                 artifactsFolder,
-			ClusterTemplate:                e2e.CAPIDockerKubeadm,
-			ClusterName:                    "clusterv3-auto-import-kubeadm",
+			ClusterTemplate:                e2e.CAPIDockerRKE2,
+			ClusterName:                    "clusterv3-auto-import-rke2",
 			ControlPlaneMachineCount:       ptr.To[int](1),
 			WorkerMachineCount:             ptr.To[int](1),
 			GitAddr:                        giteaResult.GitAddress,
 			GitAuthSecretName:              e2e.AuthSecretName,
 			SkipCleanup:                    false,
 			SkipDeletionTest:               false,
 			LabelNamespace:                 true,
-			TestClusterReimport:            true,
+			TestClusterReimport:            false,
```

Comment:
Q: while it was disabled originally in https://github.com/rancher/turtles/pull/634/files#diff-56761f4592703f66d0a3274d2c6593394805b43aefac8de23d1de3a11936eec4R87 and set to true on all other tests, why we don't need to test un-importing and re-importing the cluster?

Reply:
I was simply trying to reduce test duration for validation while in WIP. I'll change it back to the original value after the fix is double checked.